includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -110,6 +110,26 @@ public function classifyTemplate( $wikiId, $pageId, $templateType, $provider, $o
 	}
 
 	/**
+	 * Get all classified template types on given wiki with their page id as a key
+	 *
+	 * @param int $wikiId
+	 * @return array
+	 * @throws Exception
+	 * @throws \Swagger\Client\ApiException
+	 */
+	public function getTypesOnWiki( $wikiId ) {
```

Comment:
So what we actually get is an array of `page_id => type`, right? Can we call it `getClassifiedTemplatesOnWiki` or sth? `getTypes` suggests that you only get a list of types used on a given wiki. :)